_posts/2022-05-08-april-rubygems-updates.md

```diff
@@ -0,0 +1,43 @@
+---
+title: April 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in April.
+
+
+# RubyGems News
+
+In April, we released new versions of RubyGems [3.3.11](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3311--2022-04-07), [3.3.12](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3312--2022-04-20) and Bundler [2.3.11](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2311-april-7-2022), [2.3.12](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2312-april-20-2022).
+
```

Comment:
I would add a separate paragraph on top of this to highlight the addition of experimental support for rust extensions, and remove that entry from the list above.

Something like this, for example:

> As the main highlight of this month, RubyGems now has experimental builtin support for Rust extensions, thanks to the work at https://github.com/rubygems/rubygems/pull/5175.

Reply:
Got it! Thank you.